bin/poclcc.c

```diff
@@ -0,0 +1,188 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <CL/opencl.h>
+#include <assert.h>
+#include <string.h>
+
+char *kernelSource = NULL;
+char *outputFile = NULL;
+int openclDevice = CL_DEVICE_TYPE_DEFAULT;
+char *build_options = NULL;
+
+int print_help(){
+    printf("USAGE: poclcc [OPTION]... [FILE]\n");
+    printf("\n");
+    printf("OPTIONS:\n");
+    printf("\t--build_options=<options>\n");
+    printf("\t\tBuild the program with <options> options\n");
+    printf("\t-d <device_type>\n");
+    printf("\t\tSelect <device_type> as the device_type for clGetDeviceIDs." 
```

Comment:
How will we know which id will be the device we what the binary for?
Maybe we could extract a binary per device found, and add something in the name of the binary to specify for which device the binary i?